main.js

```diff
@@ -77,20 +82,23 @@ function startActivity() {
   showTimer();
 }
 
-// new function
```

Comment:
this comment is actually for selectActivityButton function. we could shave it down to: 
selectActivityButton.addEventListener("click", function(event) {
  var button = event.target;
  resetButtons();
  button.disabled = true 
  button.firstElementChild.src = `./assets/${button.value.toLowerCase()}-active.svg`
});